bin/Main.purs

```diff
@@ -174,13 +175,14 @@ main = launchAff_ do
           generateOperatorsCommand globs
 
         GenerateRc cliOptions -> do
-          rcExists <- FS.exists rcFileName
-          if rcExists then do
-            Console.error $ rcFileName <> " already exists."
-            liftEffect $ Process.exit 1
-          else do
-            let contents = Json.stringifyWithIndent 2 $ FormatOptions.toJson cliOptions
-            FS.writeTextFile UTF8 rcFileName $ contents <> "\n"
+          rcStats <- Aff.try $ FS.stat rcFileName
```

Comment:
`exists` was removed as part of the 0.15 updates, as its been recommended against by Node for years. This instead tries to stat the file, and if that throws an exception (the file is missing) then we write the file.

Reply:
Can't you use `liftEffect $ FSSync.exists`?

Reply:
That is also possible, but we can use proper async here via stat (which is also used elsewhere in this file)